internal/cluster/cluster.go

```diff
@@ -411,16 +411,38 @@ func (m *Manager) tryAssignMachineCidrDHCPMode(cluster *common.Cluster) error {
 }
 
 func (m *Manager) tryAssignMachineCidrNonDHCPMode(cluster *common.Cluster) error {
-	machineCidr, err := network.CalculateMachineNetworkCIDR(
+	primaryMachineCidr, err := network.CalculateMachineNetworkCIDR(
 		cluster.APIVip, cluster.IngressVip, cluster.Hosts, false)
```

Comment:
after https://github.com/openshift/assisted-service/pull/4755/files is merged, change this to `apivips[0]`